docs/docusaurus/versioned_docs/version-0.18/cloud/deploy_gx_agent.md

```diff
@@ -0,0 +1,247 @@
+---
+sidebar_label: 'Deploy the GX Agent'
+title: 'Deploy the GX Agent'
+id: deploy_gx_agent
+description: Deploy the GX Agent to use GX Cloud features and functionality.
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+---
+
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+To use GX Cloud features and functionality, you need to deploy the GX Agent. The GX Agent runs open source GX code in GX Cloud, and it allows you to securely access your data without connecting to it or interacting with it directly. To learn more about the GX Agent and deployment patterns, see [About GX Cloud](./about_gx.md).
+
+## Prerequisites
+
+- You have a [GX Cloud account](https://greatexpectations.io/cloud).
+
+## Get your access token and organization ID
+
+You need your access token and organization ID to deploy the GX Agent. Access tokens shouldn't be committed to version control software. 
+
+If you've used GX Cloud previously, you have your access token and organization ID, and you need to restart the GX Agent, see [Deploy the GX Agent](#deploy-the-gx-agent).
+
+<Tabs
+  groupId="copy-token"
+  defaultValue='new'
+  values={[
+  {label: 'New GX Cloud account', value:'new'},
+  {label: 'Existing GX Cloud account', value:'existing'},
+  ]}>
+<TabItem value="new">
+
+1. Sign in to GX Cloud.
+
+2. Complete the survey and then click **Continue to GX Cloud**.
+
+3. Copy and then paste the **Access token** and **Organization ID** values into a temporary file. You'll need them to deploy the GX Agent.
+
+4. Click [**Deploy the GX Agent**](#deploy-the-gx-agent).
+
+</TabItem>
+<TabItem value="existing">
+
+Use the information provided here to view your organization ID or create a new access token. This can be helpful if you've forgotten your organization ID or access token, and you need to restart the GX Agent.
+
+1. In GX Cloud, click **Settings** > **Tokens**.
+
+2. In the **User access tokens** pane, click **Create user access token**.
+
+3. In the **Token name** field, enter a name for the token that will help you quickly identify it.
+
+4. Click **Create**.
+
+5. Copy and then paste the user access token into a temporary file. The token can't be retrieved after you close the dialog.
+
+6. Click **Close**.
+
+7. Copy the value in the **Organization ID** field into the temporary file with your user access token and then save the file. 
+
+    GX recommends deleting the temporary file after you set the environment variables.
+
+8. [Deploy the GX Agent](#deploy-the-gx-agent).
+
+</TabItem>
+</Tabs>
+
+## Deploy the GX Agent
+
+The GX Agent runs open source GX code in GX Cloud, and it allows you to securely access your data without connecting to it or interacting with it directly. To learn more about the GX Agent and deployment patterns, see [Deploy the GX Agent](#deploy-the-gx-agent).
```

Comment:
The link here goes to the top of this paragraph. If we want users to keep reading, should we say something like "See below" instead?

Reply:
@annabarr this was a miss on my part. I need to update content in two locations, and the topic that is included in the builds wasn't updated. The link has been corrected to point to [About GX Cloud](https://deploy-preview-9635.docs.greatexpectations.io/docs/cloud/about_gx).